### PR TITLE
Burn Transaction Test + Refactors in Basic

### PR DESCRIPTION
**Pull request by cxloe**
[Soundtrack to this PR](https://music.youtube.com/watch?v=5syYAEc-odI)

### Motivation

Integration test coverage for Burn Transactions. Working towards standard, reusable, relatively parsable testing helper functions.

### In this PR

- Short test for burn transactions. Follows roughly the same convention as previous tests, but with less duplicated code.
- Renamed and refactored utility functions. `TestUtils` class ( to be moved out of basic later. ) 
- Namely, functions that wait for all accounts to sync, and a function that waits two blocks. Those may be redundant but it's what i ended up with.. 
- Some clearer named variables. 
- `accounts.py` is a script i ended up writing to print the ID, balance and entropy of all my imported FS accounts. 
- More typing for `fullservice.py`
- `pytest` compatibility for `basic.py` and `burn_transaction_test.py`

### Test Plan

`poetry run python burn_transaction_test.py`

### To Do

Make sure `async def wait_for_account_sync():` works right. 

### Future Work

- Get things organized! Pull out `TestUtils` into it's own file, and sort out the folder structure. 
- ( **_in a sarcastic tone of voice_** ) DATACLASSES SO I NEVER HAVE TO INVOKE `.get` EVER AGAIN IN MY LIFE I S2G IF I SEE ANOTHER `NoneType` ERR-

### CODE REVIEW

File: .internal-ci/test/fs-integration/basic.py [hunk 1]
```diff
@@ -4,67 +4,107 @@
...
         mnemonic,
         "2",  # This parameter indicates that we are using the 2nd key derivations method (mnemonics)
-        name=name
-    )  
+        name=name,
+    )
 
-    if not okay_if_already_imported:
-        assert "error" not in account.keys(),  "Failed to import account"
+    if not already_imported:
```
[the remark this answers is not in the document]
Comment (cxloe) [reply .internal-ci/test/fs-integration/basic.py]: suggestion?

**Comment by cxloe** [comment]
> I think it makes most sense for burn_transaction_test.py to live with the other integration tests and not under python-library

Folder structure is out of scope for this, will revisit in a later PR. I think `python-library/integration-tests` makes sense. CI will be able to invoke whatever tests we can to configure it to run by importing from the same place it needs to import `fullservice.py`

File: python-library/fullservice.py [hunk 2]
```diff
@@ -110,15 +110,15 @@ async def build_and_submit_transaction(
...
-        account_id,
-        amount={"value": "", "token_id": ""},
-        redemption_memo_hex="",
-        input_txo_ids="",
-        fee_value="",
-        fee_token_id="",
-        tombstone_block="",
-        max_spendable_value="",
-    ):
+        account_id: str,
```
Comment (queenbdubs) [inline python-library/fullservice.py:113]: I think you might as well add the `str` type to all the rest of the `account_id` parameters in this file.

File: .internal-ci/test/fs-integration/basic.py [hunk 3]
```diff
@@ -3,117 +3,135 @@
...
-    while network_status.get('result').get('network_block_height') != network_status.get('result').get('local_block_height'):
-        print("Sleep")
-        await asyncio.sleep(sleepy_time)
-        network_status = await fs.get_network_status()
 
-
-async def main():
-    while (await fs.get_wallet_status())['result']['wallet_status']['is_synced_all'] != True:
-        await asyncio.sleep(sleepy_time)  
+class TestUtils:
```
[the remark this answers is not in the document]
Comment (cxloe) [reply .internal-ci/test/fs-integration/basic.py:28]: I don't see any reason to have them in a class when it's a seperate file, it should behave in a similar manner when imported.

File: python-library/fullservice.py (hunk 2 above)
Comment (cxloe) [reply python-library/fullservice.py:113]: I'm actually using [monkeytype](https://github.com/Instagram/MonkeyType) to dynamically annotate the types of functions as they're called. I'll see about annotating all of the account id params though!

**Pull request merged**